.semaphore/semaphore.yml

```diff
@@ -43,6 +43,10 @@ blocks:
           - echo "max-jobs = auto" >> ~/.config/nix/nix.conf
 
       jobs:
+        - name: "Check frozen versions"
+          commands:
+            - "cabal build --dry-run all"
```

Comment:
Might need a `nix develop` - it failed on CI because it cannot find cabal

Reply:
You're right, I forgot there's no devenv there. Should be fixed now.